src/test/java/org/influxdb/InfluxDB2Test.java

```diff
@@ -0,0 +1,63 @@
+package org.influxdb;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import org.influxdb.dto.Query;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Jakub Bednar (30/08/2021 11:31)
+ */
+@RunWith(JUnitPlatform.class)
+@EnabledIfEnvironmentVariable(named = "INFLUXDB_VERSION", matches = "2\\.0")
+public class InfluxDB2Test {
+
+    private InfluxDB influxDB;
+
+    @BeforeEach
+    public void setUp() throws NoSuchFieldException, IllegalAccessException {
+        String url = String.format("http://%s:%s", TestUtils.getInfluxIP(), TestUtils.getInfluxPORT(true));
+        influxDB = InfluxDBFactory
+                .connect(url, "my-user", "my-password")
+                .setDatabase("mydb")
+                .setRetentionPolicy("autogen");
+
+        // TODO - temporary solution before release - https://github.com/influxdata/influxdb/pull/22325
```

Comment:
can we remove this as i see the PR is merged

Reply:
done